luigi_pipeline/lib/model/gcnv_mt_schema.py

```diff
@@ -0,0 +1,212 @@
+import datetime
+import re
+
+import hail as hl
+
+from lib.model.base_mt_schema import BaseMTSchema, row_annotation
+from lib.model.seqr_mt_schema import SeqrGenotypesSchema, SeqrVariantsAndGenotypesSchema
+
+from hail_scripts.computed_fields import variant_id
+
+SAMPLE_ID_REGEX = r'(.+)_v\d+_Exome_(C|RP-)\d+$'
+
+def parse_genes(gene_col: hl.expr.StringExpression) -> hl.expr.SetExpression:
+    """
+    Convert a string-ified gene list to a set()
+    """
+    return hl.set(hl.map(
+        lambda gene: gene.split(r'\.')[0],
+        hl.filter(
+            lambda gene: ~hl.set({'None', 'null', 'NA', ''}).contains(gene),
+            gene_col.split(','),
+        )
+    ))
+
+def hl_agg_collect_set_union(gene_col: hl.expr.SetExpression) -> hl.expr.SetExpression:
+    """
+    aggregate with the set union operator
+    """
+    return hl.fold(
+        lambda i, j: i | j,
+        hl.empty_set(hl.tstr),
+        hl.agg.collect(gene_col),
+    )
+
+class SeqrGCNVVariantSchema(BaseMTSchema):
+
+    @row_annotation()
+    def contig(self):
+        return self.mt.chr.replace("^chr", "")
+
+    @row_annotation()
+    def sc(self):
+        return self.mt.vac
+
+    @row_annotation()
+    def sf(self):
+        return self.mt.vaf
+
+    @row_annotation()
+    def sn(self):
+        return hl.if_else(
+            hl.is_defined(self.mt.vaf),
+            hl.int(self.mt.vac / self.mt.vaf),
+            hl.missing(hl.tint32)
+        )
+
+    @row_annotation(name='svType')
+    def sv_type(self):
+        return self.mt.svtype
+
+    @row_annotation(name='StrVCTVRE_score')
+    def strvctvre(self):
+       return self.mt.strvctvre_score
+
+    @row_annotation(name='variantId')
+    def variant_id(self):
+        return hl.format(f"%s_%s_{datetime.date.today():%m%d%Y}", self.mt.variant_name, self.mt.svtype)
+
+    @row_annotation()
+    def start(self):
+        return hl.agg.min(self.mt.sample_start)
```

Comment:
These are a little weird but duplicate the existing behavior.  Not sure we ever see different `start` and `end` over different samples?

Reply:
Got it!  I did see the sample specific starts in the genotype but I didn't implement that!  Good catch!